Code/GraphMol/MolStandardize/Metal.cpp

```diff
@@ -148,8 +148,9 @@ void MetalDisconnector::disconnect(RWMol &mol) {
       fcAfterCut = 0;
     }
     a->setFormalCharge(fcAfterCut);
-    if (fcAfterCut == -1 && a->getAtomicNum() == 6 &&
-        valenceAfterCut == static_cast<int>(a->getTotalDegree()) + 2) {
+    if (!it->second.cutBonds ||
```

Comment:
It doesn't look like there's currently a test case for the situation that would cause `it->second.cutBonds` to be zero (I think this happens when there's a dative bond).
Since you're in the code anyway, would you please add one for this?

Reply:
@greglandrum Done. There was already a test (which was failing, and which prompted me to add that additional `if` clause) with a cobalt complex, but a couple more won't hurt :-)

Reply:
@greglandrum I also refactored the tests to use scoping as I was fed up having to increment numbers every time I added a test. I hope you don't mind as I am *not* going to revert it :-)

Reply:
> @greglandrum I also refactored the tests to use scoping as I was fed up having to increment numbers every time I added a test. I hope you don't mind as I am _not_ going to revert it :-)

Na, I agree that the scoping is good. But if you were going to be cleaning up code in there, it would have been nice to take the next step and simplify as many of those test cases as possible to:
`auto m = "CCC"_smiles;`

Reply:
Doing it now. Remind me, why I started this?

Reply:
@greglandrum done.

Reply:
CODE CLEANUP IS FUN
CODE CLEANUP IS FUN
CODE CLEANUP IS FUN